### ISSUE TITLE
Allow 1 way syncing of read-only google calendars

### ISSUE THREAD

**Issue by BrettMichlitsch** [phw198/OutlookGoogleCalendarSync#1531]
My kids sports teams, church calendars, and my sports leagues all have google calendars that are read-only calendars. It would be great to be able to sync those to my outlook. Just one-way from google to outlook.

**Comment by phw198** [comment]
That should just work...! What happens when you try?

**Comment by BrettMichlitsch** [comment]
I cant create a profile without an exception occurring. I was just pressing continue through the exception then saving the profile. Everytime I load the profile I get this exception: I'm guessing maybe that why this isn't working. 
```
************** Exception Text **************
System.ArgumentException: Items collection cannot be modified when the DataSource property is set.
   at System.Windows.Forms.ComboBox.CheckNoDataSource()
   at System.Windows.Forms.ComboBox.ObjectCollection.Clear()
   at OutlookGoogleCalendarSync.Forms.Main.UpdateGUIsettings_Profile() in C:\Users\Paul\Git\OutlookGoogleCalendarSync\src\OutlookGoogleCalendarSync\Forms\MainForm.cs:line 480
   at OutlookGoogleCalendarSync.SettingsStore.Calendar.SetActive() in C:\Users\Paul\Git\OutlookGoogleCalendarSync\src\OutlookGoogleCalendarSync\SettingsStore\Calendar.cs:line 207
   at OutlookGoogleCalendarSync.Forms.Main.ddProfile_SelectedIndexChanged(Object sender, EventArgs e) in C:\Users\Paul\Git\OutlookGoogleCalendarSync\src\OutlookGoogleCalendarSync\Forms\MainForm.cs:line 1073
   at System.Windows.Forms.ComboBox.OnSelectedIndexChanged(EventArgs e)
   at System.Windows.Forms.ComboBox.WmReflectCommand(Message& m)
   at System.Windows.Forms.ComboBox.WndProc(Message& m)
   at System.Windows.Forms.Control.ControlNativeWindow.OnMessage(Message& m)
   at System.Windows.Forms.Control.ControlNativeWindow.WndProc(Message& m)
   at System.Windows.Forms.NativeWindow.Callback(IntPtr hWnd, Int32 msg, IntPtr wparam, IntPtr lparam)
```
If I press sync for that profile I get this exception from the log files:
```
Problem encountered during synchronisation.
2022-08-31 11:58:37,347 ERROR  1 OutlookGoogleCalendarSync.Console [283] -  Object reference not set to an instance of an object.
2022-08-31 11:58:37,369 ERROR  1 OutlookGoogleCalendarSync.OGCSexception [28] -  System.NullReferenceException: Object reference not set to an instance of an object.
2022-08-31 11:58:37,370 ERROR  1 OutlookGoogleCalendarSync.OGCSexception [21] -  Code: 0x80004003,-2147467261; Location: StartSync() in Calendar.cs:222:60
2022-08-31 11:58:37,370 ERROR  1 OutlookGoogleCalendarSync.OGCSexception [65] -     at OutlookGoogleCalendarSync.Sync.Engine.Calendar.StartSync(Boolean manualIgnition, Boolean updateSyncSchedule) in C:\Users\Paul\Git\OutlookGoogleCalendarSync\src\OutlookGoogleCalendarSync\Sync\Calendar.cs:line 222
   at OutlookGoogleCalendarSync.Sync.Engine.Start(Boolean manualIgnition, Boolean updateSyncSchedule) in C:\Users\Paul\Git\OutlookGoogleCalendarSync\src\OutlookGoogleCalendarSync\Sync\Engine.cs:line 212
   at OutlookGoogleCalendarSync.Sync.Engine.Sync_Requested(Object sender, EventArgs e) in C:\Users\Paul\Git\OutlookGoogleCalendarSync\src\OutlookGoogleCalendarSync\Sync\Engine.cs:line 174
   at OutlookGoogleCalendarSync.Forms.Main.Sync_Click(Object sender, EventArgs e) in C:\Users\Paul\Git\OutlookGoogleCalendarSync\src\OutlookGoogleCalendarSync\Forms\MainForm.cs:line 668
```

**Comment by phw198** [comment]
Yes, that first error at least will be fixed on #1480 - let me know if it works after applying that hotfix.
